README.md

```diff
@@ -1,11 +1,11 @@
 # CLFFT
 
-**Build status**: [![][gitlab-img]][gitlab-url]
+**Build status (deprecated)**: [![][gitlab-img]][gitlab-url]
```

Comment:
If the badge is outdated I think we should just remove it. It is not very helpful to see a deprecated badge.